### PR TITLE
Issue 37324: Logging updates in ListImporter

### PR DESCRIPTION
**Pull request by RosalineP**
Hi Binal!

Looking over your changes and the file, it looked like the only changes needed were to keep a successfulLists counter, and update the logging messages to Adam's suggestions, since you'd already implemented skipping processing if a data file is missing during list archive import.

I did basic manual tests on this and the new logging statements are as expected!

### CODE REVIEW

File: list/src/org/labkey/list/model/ListImporter.java [hunk 1]
```diff
@@ -318,11 +319,13 @@ public void processMany(VirtualFile listsDir, Container c, User user, List<Strin
...
                 failedLists++;
+            } else {
+                successfulLists++;
             }
         }
 
-        int size = lists.size() - failedLists;
-        log.info(size + " list" + (1 == size ? "" : "s") + " imported" + (failedLists == 0 ? ", all successfully" : ""));
+        log.info(successfulLists + " list" + (1 == successfulLists ? "" : "s") + " imported successfully");
+        log.info(failedLists + " list" + (1 == failedLists ? "" : "s") + " failed to import");
```
Comment (labkey-bpatel) [inline list/src/org/labkey/list/model/ListImporter.java]: log message on failedLists is already present on line 331, so this one can be removed. Other than that, the rest looks good. thanks!

Comment (RosalineP) [reply list/src/org/labkey/list/model/ListImporter.java]: Fixed!

File: list/src/org/labkey/list/model/ListImporter.java [hunk 2]
```diff
@@ -325,7 +325,6 @@ public void processMany(VirtualFile listsDir, Container c, User user, List<Strin
         }
 
         log.info(successfulLists + " list" + (1 == successfulLists ? "" : "s") + " imported successfully");
-        log.info(failedLists + " list" + (1 == failedLists ? "" : "s") + " failed to import");
         if (failedLists > 0)
         {
             log.warn(failedLists + " list" + (1 == failedLists ? "" : "s") + " failed to import");
```
Comment (labkey-adam) [inline list/src/org/labkey/list/model/ListImporter.java]: Not critical, of course, but using `StringUtilsLabKey.pluralize(failedLists, "list")` would simplify this.

Comment (RosalineP) [reply list/src/org/labkey/list/model/ListImporter.java]: Good to know, thank you! :) 
Update made.

**Pull request merged**